src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(double side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void toDraw() {
```

Comment:
The method name `toDraw` does not follow Java naming conventions for methods. It should be an action verb, such as `draw`. Also, ensure that the method name matches the one in the `Drawable` interface.